docs/src/main/asciidoc/datasource.adoc

```diff
@@ -88,25 +89,32 @@ quarkus.datasource.password=<your password>
 quarkus.datasource.reactive.url=postgresql:///your_database
 quarkus.datasource.reactive.max-size=20
 ----
-<1> As specified above this is optional.
+<1> This is optional.
+
+== How to configure datasources
 
-== Default datasource
+The following section describes the configuration for single or mutual datasources.
+For simplicity, we will reference a single datasource as the default (unnamed) datasource.
```

Comment:
That should be the "for simplicity" part. But yeah maybe it can be made more clear

For simplicity, for single-datasource application, we will not give the datasource a name, so we will use the so-called default datasource..

Don't know if this is better?!

Reply:
For simplicity, I would just omit this sentence :) After all, we're just describing how we think a single-datasource app should be configured, as well as how we think a mult-datasource app should be configured. No need to explain further IMO.